scheduler/data-flow/build.gradle.kts

```diff
@@ -24,39 +24,40 @@ repositories {
 
 dependencies {
     implementation("com.natpryce:konfig:1.6.10.0")
-    implementation("io.klogging:klogging-jvm:0.5.11")
-    implementation("io.klogging:slf4j-klogging:0.5.11")
+    implementation("io.klogging:klogging-jvm:0.5.14")
+    implementation("io.klogging:slf4j-klogging:0.5.14")
 
     // Kafka
     implementation("org.apache.kafka:kafka-streams:7.6.1-ccs")
     testImplementation("org.apache.kafka:kafka-streams-test-utils:7.6.1-ccs")
 
     // gRPC
     implementation("io.grpc:grpc-kotlin-stub:1.4.1")
-    implementation("io.grpc:grpc-stub:1.64.0")
-    implementation("io.grpc:grpc-protobuf:1.64.0")
-    runtimeOnly("io.grpc:grpc-netty-shaded:1.64.0")
-    implementation("com.google.protobuf:protobuf-java:3.25.3")
-    implementation("com.google.protobuf:protobuf-kotlin:3.25.3")
+    implementation("io.grpc:grpc-stub:1.65.0")
+    implementation("io.grpc:grpc-protobuf:1.65.0")
+    runtimeOnly("io.grpc:grpc-netty-shaded:1.65.0")
+    implementation("com.google.protobuf:protobuf-java") {
+        version {
+            strictly("[4.27.2,)")
+            prefer("4.27.2")
```

Comment:
Here, the major version "4" is java-specific, while `{minor}.{patch}` reference the protobuf version (27.2).